packages/icons-workflow/package.json

```diff
@@ -60,8 +60,6 @@
         "case": "^1.6.1",
         "cheerio": "^1.0.0-rc.2",
         "fast-glob": "^3.2.12",
-        "fs": "^0.0.1-security",
```

Comment:
fs and path are bundled with node and don't need to be pulled in explicitly (in fact it can be a risk).